packages/react-vapor/src/components/listBox/ListBox.tsx

```diff
@@ -6,6 +6,11 @@ import {mod} from '../../utils/DataStructuresUtils';
 import {callIfDefined} from '../../utils/FalsyValuesUtils';
 import {IItemBoxProps, ItemBox} from '../itemBox/ItemBox';
 
+export interface IITemBoxPropsWithIndex {
+    index?: number;
+    item: IItemBoxProps;
+}

```

Comment:
Wouldn't it be simpler to use a simple type intersection?
```ts
{index?: number} & IItemBoxProps
```

Reply:
That way you wouldn't have to use `itemWithIndex.item.something`, and could simply use `itemWithIndex.active` instead.